build.sbt

```diff
@@ -12,21 +11,26 @@ lazy val jvmSdkVersion = "1.1.0"
  */
 
 lazy val `commercetools-sunrise-payment` = (project in file("."))
-  .aggregate(`common`, `payone`)
+  .aggregate(`common`, `payone-adapter`)
   .settings(javaUnidocSettings ++ commonSettings : _*)
+  .settings(
+      libraryDependencies ++= Seq (
+        "com.novocode" % "junit-interface" % "0.11",
+        "org.assertj" % "assertj-core" % "3.4.1"
+      )
+  )
+  .dependsOn(`common`, `payone-adapter`)
 
 lazy val `common` = project
   .configs(IntegrationTest)
   .settings(commonSettings ++ commonTestSettings : _*)
   .settings(
     libraryDependencies ++= Seq(
-      "com.commercetools.sdk.jvm.core" % "commercetools-models" % jvmSdkVersion,
-      "com.commercetools.sdk.jvm.core" % "commercetools-java-client" % jvmSdkVersion,
-      "com.google.code.findbugs" % "jsr305" % "3.0.0"
```

Comment:
In Sunrise we try to put the nullable parameters/fields with the `@Nullable` annotation, so that developers get an IDE warning if they try to access it without a null check. When it does not have this annotation, it is supposed to be not null. We need this because of Java 8's `Optional` limitations.